packages/salesforcedx-vscode-replay-debugger/src/states/frameExitState.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { LogContext } from '../core/logContext';
+import { DebugLogState } from './debugLogState';
+
+export class FrameExitState implements DebugLogState {
```

Comment:
What happens if you don't see a corresponding METHOD_ENTRY and METHOD_EXIT (due to log truncation)?

METHOD_ENTRY method1()
METHOD_ENTRY method2()
...truncation... // exited method2
METHOD_EXIT method1() <- does this one know to pop off both method1 and method2